src/transactions/feed/EarnFeedItem.tsx

```diff
@@ -0,0 +1,175 @@
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { EarnEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import TokenDisplay from 'src/components/TokenDisplay'
+import Touchable from 'src/components/Touchable'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { getDynamicConfigParams } from 'src/statsig'
+import { DynamicConfigs } from 'src/statsig/constants'
+import { StatsigDynamicConfigs } from 'src/statsig/types'
+import { Colors } from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import variables from 'src/styles/variables'
+import TransactionFeedItemImage from 'src/transactions/feed/TransactionFeedItemImage'
+import { EarnClaimReward, EarnDeposit, EarnWithdraw } from 'src/transactions/types'
+
+interface DescriptionProps {
+  transaction: EarnWithdraw | EarnDeposit | EarnClaimReward
+}
+
+function Description({ transaction }: DescriptionProps) {
+  const { t } = useTranslation()
+  const { providerName } = getDynamicConfigParams(
```

Comment:
probably a rare edge case, but in case there are multiple providers in the future, and someone is on the app version that gets the name from the dynamic config, and have earn txs from a different provider, they'll see the Aave name.. maybe one option would be to just filter by provider id (hardcoded aave-v3) for now to be extra safe?

Reply:
I think we'll have a similar issue with the bottom sheet so I think its fine for now and when we have multiple providers we can have a more robust solution.